server/bleep/src/webserver.rs

```diff
@@ -54,7 +54,8 @@ pub async fn start(app: Application) -> anyhow::Result<()> {
         .route("/hoverable", get(hoverable::handle))
         .route("/token-info", get(intelligence::handle))
         // misc
-        .route("/search", get(semantic::complex_search))
+        .route("/search/code", get(search::code_search))
+        .route("/search/path", get(search::path_search))
```

Comment:
can we clarify somewhere that these are semantic code search and fuzzy path search? if these endpoints are studio specific, maybe we should use a new route altogether, `studio/xxx`. @calyptobai what do you think?

Reply:
These endpoints are not strictly studio-specific. They can also be used to debug the `Agent` flow (they simulate the `code` and `path` tools).